web_widget_kanban_many2one/static/src/js/web_widget_kanban_many2one.js

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2019 Sunflower IT <www.sunflowerweb.nl>
+ * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+ */
+
+odoo.define('web_widget_kanban_many2one.kanban_editable', function(require){
+
+    var AbstractField = require('web.AbstractField');
+    var core = require('web.core');
+    var field_registry = require('web.field_registry');
+
+    var _t = core._t;
+
+    var many2one_kanban = AbstractField.extend({
```

Comment:
I think that better use "KanbanFieldMany2One" and use an option like "editable='true'"

Reply:
"editable=true" in xml?

Reply:
@Tardo I disagree; this widget is meant to add editable functionality also in non-editable mode, just like for example the priority stars widget can do when you are in read-only kanban mode and click one of the stars. This is one of our customer's requirements, for faster many2one editing.

@Kkamaa What I think he meant is that we implement this widget as the editable version of the existing many2one widget in the Odoo source, using the "editable" boolean in the rendering function; this is not in XML but just in javascript. You can check some other widgets that use this functionality such as this one: https://github.com/OCA/OCB/blob/12.0/addons/web/static/src/js/fields/relational_fields.js#L1487

Reply:
@thomaspaulb about this?

Reply:
@Kkamaa I dont agree with Tardo, so unless he replies with a convincing argument, we dont work on this.